Tests/UnitTests/Paywalls/PaywallDataTests.swift

```diff
@@ -122,6 +122,25 @@ class PaywallDataTests: BaseHTTPResponseTest {
         expect(esConfig.title) == "Tienda"
     }
 
+    func testLocalizedConfigurationFallsBackToLanguageWithDifferentRegion() throws {
+        let paywall: PaywallData = try self.decodeFixture("PaywallData-Sample1")
+
+        let enConfig = try XCTUnwrap(paywall.localizedConfiguration(for: [
+            .init(identifier: "en_IN"),
+            .init(identifier: "en-IN"),
+            .init(identifier: "en-IN").removingRegion
```

Comment:
The fact that this is included in `Paywall.localizedConfiguration` is tested below.